packages/app/src/app/pages/Dashboard/queries.js

```diff
@@ -19,6 +19,11 @@ const SANDBOX_FRAGMENT = gql`
     privacy
     screenshotUrl
 
+    author {
```

Comment:
It might be better to create a separate fragment for this, or extend the fragment. For some users we load this fragment 500+ times in one query on "My Sandboxes", every addition can be a couple kbs and seconds of extra response time.

Reply:
I don't know if Absinthe has something similar, but I know with Apollo Server you can set up response deduplication, which would help to optimize the response payload size if that's a concern.

Reply:
Hmm, now that I think of it, it also is probably handled by gzip for a big part. It would be good to remove this nonetheless from the fragment I think, since we only use it for one query.